docs/index.md

```diff
@@ -402,18 +402,17 @@ getting_started/core_use_cases
 :caption: User Guide
 :hidden:
 
-User Guide <userguide>
-Environment Setup <environment_setup>
-Basics <auto_examples/basics/index>
-Control Flow <auto_examples/control_flow/index>
-Type System <auto_examples/type_system/index>
-Testing <auto_examples/testing/index>
-Containerization <auto_examples/containerization/index>
-Image Spec <auto_examples/image_spec/index>
-Remote Access <auto_examples/remote_access/index>
-Production Config <auto_examples/deployment/index>
-Extending Flyte <auto_examples/extend_flyte/index>
-Example Contribution Guide <contribute>
+📖 User Guide <userguide>
```

Comment:
These emojis look great!

My one concern I have tho is, for visual consistency, I would expect the tutorials and integrations to have an emoji per entry, which seems like a difficult thing to pull off, esp for integrations.

<img width="358" alt="image" src="https://github.com/flyteorg/flytesnacks/assets/2816689/848eaf07-c709-44fd-9ba8-05be13a53394">

<img width="343" alt="image" src="https://github.com/flyteorg/flytesnacks/assets/2816689/84d8e4eb-0c22-4465-8daf-b8e3b32b653f">

Maybe this is just me being nitpicky about consistency tho 🤷‍♂️ maybe we can do emojis for Tutorials but not for Integrations, which you might be planning on collapsing/re-organizing too.